From a52785b4e47b503d0b4b3f2f4175be097f8b8fcc Mon Sep 17 00:00:00 2001
From: Ajay Kaher <akaher@vmware.com>
Date: Sun, 12 Feb 2023 17:40:45 +0000
Subject: [PATCH] add saveenv in bootcmd

This patch is to add saveenv to the boot command. So, while
booting it will save the u-boot env variables in uboot.env file.
---
 include/config_distro_bootcmd.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/config_distro_bootcmd.h b/include/config_distro_bootcmd.h
index 12486c86..d6f95912 100644
--- a/include/config_distro_bootcmd.h
+++ b/include/config_distro_bootcmd.h
@@ -533,6 +533,7 @@
 		BOOTENV_SET_IDE_NEED_INIT                                 \
 		BOOTENV_SET_VIRTIO_NEED_INIT                              \
 		BOOTENV_SET_EXTENSION_NEED_INIT                           \
+		"saveenv;"                                                \
 		"for target in ${boot_targets}; do "                      \
 			"run bootcmd_${target}; "                         \
 		"done\0"
-- 
2.19.0

